cmake/Modules/FindClangFormat.cmake

```diff
@@ -12,30 +12,19 @@
 #
 # .. code-block:: cmake
 #
-# find_package(ClangFormat) 
+# find_package(ClangFormat)
 # if(ClangFormat_FOUND)
 # message("clang-format executable found: ${ClangFormat_EXECUTABLE}\n"
-#         "version: ${ClangFormat_VERSION}") 
+#         "version: ${ClangFormat_VERSION}")
 # endif()
 
 find_program(ClangFormat_EXECUTABLE
              NAMES
-                   clang-format-9
-                   clang-format-9.0
-                   clang-format-8
-                   clang-format-8.0
-                   clang-format-7
-                   clang-format-7.0
-                   clang-format-6.0
-                   clang-format-5.0
-                   clang-format-4.0
-                   clang-format-3.9
-                   clang-format-3.8
-                   clang-format-3.7
-                   clang-format-3.6
-                   clang-format-3.5
-                   clang-format-3.4
-                   clang-format-3.3
+             # 3.3-3.9, 4.0-9.0 have been removed because they aren't needed by PCL
```

Comment:
I think the idea was that `FindClangFormat.cmake`  is a "universal" finder script that finds  `clang-format` and detects its version (whatever that might be). Then in our `clang-format.cmake` we find the specific version we support: 

https://github.com/PointCloudLibrary/pcl/blob/908441c28a7d8b8a85608b3ae8ce0397889c125e/cmake/clang-format.cmake#L1

With your changes, the end result is the same on a system that has supported `clang-format` installed. However, on a system with an older `clang-format`, this new approach will just fail to detect anything (which may be confusing to the user), whereas the old approach will detect an outdated `clang-format` and provide a reasonable diagnostics to the user "look man, you have `clang-format-8`, sure, but that's not enough for us".

Reply:
Got it. I'll revert